### PR TITLE
Add api key

### PR DESCRIPTION
**Pull request by apicl**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
First add the api attributs into install/kubernetes/templates/istio-mixer.yaml.tmpl, then run ./install/updateVersion.sh

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by geeknoid** [comment]
I don't know the purpose of any of these files, so I'll pass on reviewing this :-)

File: install/kubernetes/istio-auth.yaml [hunk 1]
```diff
@@ -673,6 +673,8 @@ spec:
       valueType: STRING
     request.auth.presenter:
       valueType: STRING
+    request.api_key:
+      valueType: STRING
```
Comment (manlinl) [inline install/kubernetes/istio-auth.yaml]: Do you need change the template instead?

https://github.com/istio/istio/tree/master/install/kubernetes/templates

File: mixer/OWNERS [hunk 2]
```diff
@@ -8,10 +8,12 @@ reviewers:
   - ozevren
   - sebastienvas
   - ZackButcher
+  - apicl
```
Comment (manlinl) [inline mixer/OWNERS]: Need merge with your OWNERS file change to remove this file from PR.

File: install/kubernetes/istio-auth.yaml (hunk 1 above)
Comment (apicl) [reply install/kubernetes/istio-auth.yaml]: This is auto generated by install/updateVersion.sh

**Comment by apicl** [comment]
PTAL

**Comment by geeknoid** [comment]
I'd like someone familiar with all these files to comment first before approving...

**Pull request merged**